api/data_test.go

```diff
@@ -23,6 +23,7 @@ var testCacheID = "a1b2c3d4e5f67890123456789abcdef0"
 var baseURL = "http://localhost:29100/v1/cache-times/"
 var staticTime = time.Date(2024, time.January, 1, 0, 0, 0, 0, time.UTC)
 var staticTimePtr = &staticTime
+var exampleCollectionID = "test-1a19e3462937d85804752375daa00ba41d1b6625d396f21000e3c4571ebf2606"
```

Comment:
I'd consider renaming this to `testCollectionID`, since we have another variable called `testID`.